cosmo_tester/framework/test_hosts.py

```diff
@@ -341,7 +342,7 @@ def assign(
         self.node_instance_id = node_instance_id
         self.deployment_id = deployment_id
         self.server_id = server_id
-        self.install_config = {
+        self.basic_install_config = {
```

Comment:
Does the base one need to be an instance variable? If it wasn't then I think the copy.deepcopy could probably be safely removed too?

Reply:
Right, I haven't explained in the description of the PR the purpose of reverting the `install_config`. 
When we install a 3 nodes cluster, the same node is used for rabbitmq, DB, and Manager. During the bootstrap of each service (rabbitmq, DB, Manager), the `node.install_config` is updated with the values of the service currently being installed, but also keeps the values of the previous one. This means that after the cluster bootstrap, the `install_config` of each node has **all** values from each config file. This is why we need to revert the `install_config` of each node after installing each service in the 3 nodes case. 

We could delete the `basic_install_config` and put it in a global variable, but I don't think it would make things easier, i.e. we would still need `copy.deepcopy`.

Note: We could also revert the `install_config` by default for each node in the [run_cluster_bootstrap function](https://github.com/cloudify-cosmo/cloudify-system-tests/blob/1995acb59585eb1ced34fb23c6c0cd12f01a0c70/cosmo_tester/test_suites/cluster/conftest.py#L248), but I thought it might cause issues.

Reply:
Hmm. OK, let's go with this for the moment and revisit it later once we do some more three node tests and get a feel for what's warty and what's necessary.